batch_convert.qml

```diff
@@ -709,7 +754,7 @@ MuseScore {
         }
 
         onAccepted: {
-            Qt.quit()
+            //Qt.quit()
```

Comment:
Huh?

Reply:
That "Qt.quit()" despite having no effect is not suitable and not correct.
No suitable because it would close the plugin, which is not desired, so that the user can launch another conversion after a first one.
Not correct because it would close the entire Qt system and not only the plugin.

Reply:
True, but why not `workDialog.parent.Window.window.close();`?

Reply:
See also 120d34c

Reply:
Actually not needed. Pressing the button closes the dialog without requiring anything else to be done. I guess this is because the button is a StandardButton.Ok, although I don't find any documentation that would confirm this.

Reply:
OK. Then let's remove the commented out line entirely